Core/Core/Dashboard/View/CourseCard.swift

```diff
@@ -74,31 +55,89 @@ struct CourseCard: View {
                 .offset(x: 8, y: 8)
                 .zIndex(1)
 
-            customizeButton
-                .offset(x: width - 44, y: 0)
+            if !isWideLayout {
+                customizeButton
```

Comment:
Unifying this to always be in the corner related to the courseImage might suit all design scenarios.